src/os/bluestore/BlueStore.cc

```diff
@@ -5692,6 +5713,15 @@ void BlueStore::_wctx_finish(
       if (compressed) {
         txc->statfs_delta.compressed_allocated() -= e.length;
       }
+      if (l.blob < 0) {
```

Comment:
Why doing that inside the loop over released pextents? IMHO this can be done at line 5705 too.